ipatests/test_xmlrpc/test_add_remove_cert_cmd.py

```diff
@@ -352,3 +376,88 @@ def delete_entity(cls):
             api.Command.host_del(TestCertManipCmdHost.entity_pkey)
         except errors.NotFound:
             pass
+
+
+@pytest.mark.tier1
+class TestCertManipIdOverride(XMLRPC_test):
+    entity_subject = u'testuser'
+    entity_principal = u'testuser'
+
+    cert_add_cmd = api.Command.idoverrideuser_add_cert
+    cert_del_cmd = api.Command.idoverrideuser_remove_cert
+
+    def del_cert_from_idoverride(self, username, view_name, cert):
```

Comment:
Since the method operates on the idview entry, it makes sense to move it to the tracker to be available for other tests as well. Please move it to tracker.